src/integrationTest/java/com/github/qwazer/markdown/confluence/Version.java

```diff
@@ -0,0 +1,26 @@
+package com.github.qwazer.markdown.confluence;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+
+public final class Version {
+
+    public static final String PROJECT_VERSION;
+
+    static {
```

Comment:
.properties file can also be parsed with [Properties](https://docs.oracle.com/javase/8/docs/api/java/util/Properties.html) class.

Reply:
True, and would probably be much easier this way - let me change that.